magento/pwa-studio
# [Bugfix] Storybook command fails

## Description

Updates mostly Storybook configuration files to prevent the Storybook command from failing in the core project and scaffolded projects.

## Related Issue







Closes PWA-2230

## Acceptance

Any developer

### Verification Stakeholders

Any developer

### Specification




## Verification Steps





1. Run storybook command: `yarn storybook:venia`
2. Verify storybook starts
3. Verify the Header story looks correct (i.e. tailwind styles have been applied)
4. Create a new scaffolded project:

```
DEBUG_PROJECT_CREATION=1 yarn buildpack create-project scaffold-test --template "packages/venia-concept" --name "scaffold-test" --author "Scaffold Test" --backend-url "https://master-7rqtwti-mfwmkrjfqvbjk.us-4.magentosite.cloud/" --npm-client "yarn"
```
5. Run storybook command in the scaffolded project: `yarn run storybook`
6. Verify storybook starts
7. Verify the Header story looks correct (i.e. tailwind styles have been applied)

#### Test scenario(s) for direct fix/feature

1. Verify storybook starts and Header test looks correct




#### Test scenario(s) for any existing impacted features/areas




1. Verify storybook still runs

#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?

No




#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
